### ISSUE TITLE
Activating 2FA produces a SQL error

### ISSUE THREAD

**Issue by GeoffreyFrogeye** [firefly-iii/firefly-iii#2461]
**Bug description**
I am running Firefly III version 6f75e0df3cf66edf2d3a4922a31408beba92fc1b, and my problem is:

Activating 2FA produces a 500 error with the following message:

```
SQLSTATE[42703]: Undefined column: 7 ERROR: column "mfa_secret" of relation "users" does not exist LINE 1: update "users" set "mfa_secret" = $1, "updated_at" = $2 wher... ^ (SQL: update "users" set "mfa_secret" = K52QOFAG4IWLIMCH, "updated_at" = 2019-08-21 22:18:54 where "id" = 1)
```

[Full error page with trace](https://geoffrey.frogeye.fr/files/GKG3AwwsjjvvQk7y/2faActivation.html)

(don't worry about the secret being public, this is a test instance)

**Steps to reproduce**

- Go to the profile page
- Click on "Activate 2FA" (roughly translated back to english if you don't mind)
- Enter a valid code, and validate

**Extra info**

Debug information generated at 2019-08-21 22:27:49 Europe/Paris for Firefly III version **4.8.0.2**.

| Variable | Content |
| --- | --- |
| FF version | 4.8.0.2 |
| FF API version | 0.10.0 |
| App environment | local |
| App debug mode | true |
| App cache driver | file |
| App logging | , daily |
| PHP version | 7.3.8 |
| Display errors | Off |
| Session start | 2019-08-01 00:00:00 |
| Session end | 2019-08-31 23:59:59 |
| Session first | 2015-05-27 00:00:00 |
| Error reporting | ALL errors |
| Host | Linux |
| Interface | fpm-fcgi |
| UserID | 1 |
| Attempt at "en" | false |
| Attempt at "English" | false |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| Login provider |   |
| Storage disks | local-upload  |
| Using Sandstorm? | no |
| Is Sandstorm (.env) | false |
| Is Docker (.env) | false |
| bunq uses sandbox | false |
| Trusted proxies (.env) |  |
| User agent | Mozilla/5.0 (X11; Linux x86_64; rv:68.0) Gecko/20100101 Firefox/68.0 |
| Loaded extensions | Core, date, libxml, openssl, pcre, zlib, ctype, dom, fileinfo, filter, hash, json, mbstring, pcntl, SPL, PDO, session, posix, readline, Reflection, standard, SimpleXML, Phar, tokenizer, xml, xmlreader, xmlwriter, mysqlnd, cgi-fcgi, bcmath, calendar, curl, exif, gd, gettext, iconv, imap, intl, ldap, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, soap, sockets, sqlite3, zip, apcu, igbinary, imagick, redis, Zend OPcache |

**Comment by JC5** [comment]
You'll have to double check if all the migrations took, because there's code to create that field.

**Comment by GeoffreyFrogeye** [comment]
I did double check, it did said that all the migrations took place. However, I tried again with a fresh installation and a migration from a 4.7.17 install and it did work. Now that I think about it, the installation with the problem might also be the one where I had [the PostgreSQL issues](https://github.com/firefly-iii/firefly-iii/issues/2306), so maybe a migration was marked as done when it shouldn't have been during this period.

So it's fine then, sorry for bothering!
